components/auth/AppleAuth.tsx

```
@@ -1,72 +1,75 @@
+              provider: "apple",
+              token: credential.identityToken,
+            });
+
+            if (error) {
+              throw error;
+            }
+
+            // Update user metadata if sign-in was successful
+            if (data.user) {
+              const metadataResult = await updateUserMetadata({
+                firstName: credential.fullName!!.givenName,
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)